python/ray/autoscaler/azure/example-full.yaml

```diff
@@ -42,6 +42,10 @@ provider:
     resource_group: ray-cluster
     # set subscription id otherwise the default from az cli will be used
     # subscription_id: 00000000-0000-0000-0000-000000000000
+    # set unique subnet mask or a random mask will be used
+    # subnet_mask: 10.0.0.0/16
+    # set unique id for resources in this cluster or a random id will be used
```

Comment:
Same as in the schema/template above, can you provide a little more detail about how we derive it? e.g. `If not set, the default unique id will be derived from the cluster name and resource group.`